### ISSUE TITLE
Open a new file in a new tab utilizing bookmarks

### ISSUE THREAD

**Issue by normenmueller** [vim-ctrlspace/vim-ctrlspace#174]
Hi, brand new to this promising plug-in, I've the following question:

I open up a new tab (`tabe`); go to one of my bookmarks (`C-Space` | `b` | select a bookmark | `CR`). Then CS states "CWD is now: ...". Next, I want to open a file out of this new CWD, BUT I only get the file list of the previously opened tab.

Even `:e <tab><tab>` doesn't give me a list of the new CWD. Instead `:pwd` tells me a different CWD than stated by CtrlSpace.

What am I missing here?

**Comment by normenmueller** [comment]
Just disabled `set autochdir` and bookmarks, in particular, CWD seems to work...

**Comment by normenmueller** [comment]
Well, actually disabling `set autochdir` isn't such a good idea. Is there any other solution?

**Comment by Konfekt** [comment]
Does https://github.com/vim-ctrlspace/vim-ctrlspace/pull/168 fix anything ?

**Comment by normenmueller** [comment]
@Konfekt Unfortunately not. I have `autochdir` enabled. I open up a fresh MacVim. Then use my CtrlSpace bookmark to go to one of my projects. Then I `:tabe` and, again, use my CtrlSpace bookmarks to go to another project. However, when initiating `:e` to open a file of the second project, I'm still in the directory of the the first project.

**Comment by Konfekt** [comment]
Yeah, CtrlSpace does not play well with autochdir. That said of I understand @ctrlspace correctly, CtrlSpace is meant to only open one project in one Vim instance.
